system_files/shared/usr/libexec/ublue-motd

```diff
@@ -23,6 +23,18 @@ if [[ -f "$TIP_FILE" ]]; then
 	fi
 
 	TIP_ESCAPED=$(escape "$TIP")
+fi
+
+# check for secure boot key
+KEY_WARN=""
+FINGERPRINT="2B:E9:91:E3:B1:B5:40:70:F4:3D:80:BB:13:EB:C6:57:E5:A3:78:0D"
+mokutil --list-enrolled | grep -q $FINGERPRINT
```

Comment:
I don't think this actually works.

Example:
```
$ FINGERPRINT="2B:E9:91:E3:B1:B5:40:70:F4:3D:80:BB:13:EB:C6:57:E5:A3:78:0D"
$ mokutil --list-enrolled |grep $FINGERPRINT
```

Notice there is no output because the grep match failed...

I think you want something like this:
```
$ mokutil --list-enrolled --verbose-listing|grep -i $FINGERPRINT
SHA1 Fingerprint: 2b:e9:91:e3:b1:b5:40:70:f4:3d:80:bb:13:eb:c6:57:e5:a3:78:0d
```

Reply:
it does work, but not by comparing output.  when grep fails to find a match, it gives an exit code of 1.  so this just compares the exit code to see if it matches or not.

I've tested it in a VM and it works well.  if there's a better option, I'm open to changing it

Reply:
I understand the `grep -q` usage, my point is it will always think the key is not enrolled because of the output from `mokutil --list-enrolled` vs `mokutil --list-enrolled --verbose-listing` 

```
$ mokutil --list-enrolled 
2bb010e24d fedoraca
2be991e3b1 ublue kernel
```
The output above will never have the fingerprint contents. 

Adding `--verbose-listing` does add the fingerprint output to the listing, however, with a lowercased fingerprint, which won't match what you have here.

Reply:
thats not what the included mokutil command produces
```
mokutil --list-enrolled | grep Finger
SHA1 Fingerprint: 2b:b0:10:e2:4d:94:c6:32:24:58:89:ba:aa:9e:d0:f3:d5:ef:1f:68
SHA1 Fingerprint: 2b:e9:91:e3:b1:b5:40:70:f4:3d:80:bb:13:eb:c6:57:e5:a3:78:0d
```

Reply:
```
$ which mokutil --version
GNU which v2.21, Copyright (C) 1999 - 2015 Carlo Wood.
GNU which comes with ABSOLUTELY NO WARRANTY;
This program is free software; your freedom to use, change
and distribute this program is protected by the GPL.


$ /usr/bin/mokutil --list-enrolled | grep Finger

```

Does Bluefin somehow modify mokutil default behavior?

Reply:
![image](https://github.com/user-attachments/assets/200e04e8-c855-4cb5-80b0-ba5dacc679f3)

Reply:
@jardon and I discovered in a side chat that the difference here is Fedora 39 has mokutil version 0.6.0 and Fedora 40 has version 0.7.1. 

And the outputs differ.